File: fixtures/scale/csharp/medium/Repl/HelpCommand.cs
namespace Calc.Repl
{
    public static class HelpCommand
    {
        public static string Run(ReplContext context, string[] parts)
        {
            var names = string.Join(", ", context.Functions.Names());
            return string.Join("\n",
                "commands: :help :vars :history :precision N :angle MODE :quit",
                "functions: " + names,
                "assign with  name = expression");
        }
    }
}

File: fixtures/scale/php/medium/src/Repl/HelpCommand.php
<?php

namespace Calc\Repl;

class HelpCommand
{
    public static function run(ReplContext $context, array $parts): string
    {
        $names = implode(', ', $context->functions->names());
        return implode("\n", [
            'commands: :help :vars :history :precision N :angle MODE :quit',
            'functions: ' . $names,
            'assign with  name = expression',
        ]);
    }
}
